woocommerce/woocommerce-android
# [Screenshot Automation] IPP screenshot

Part of #6833
⚠️ Please don't merge

### Description
This PR adds a screenshot test for in-person payments dialog.

### Testing instructions
You can either execute `bundle exec fastlane take_screenshots locales:en-US` to check screenshots, or leave testing until the last PR 😄.

### Images/gif
|  |  |
| - | - |
| ![image](https://user-images.githubusercontent.com/1657201/177608846-59ba6a64-5b46-4d3f-8d25-30dfeaf536b5.png) | ![image](https://user-images.githubusercontent.com/1657201/177608858-5efbc49c-daad-47e6-9ae7-4bc1ada6b1d6.png) |
